sinks/sinks_service_test.go

```diff
@@ -118,18 +118,49 @@ func TestUpdateSink(t *testing.T) {
 
 	noConfig := sk
 	noConfig.Config = make(map[string]interface{})
-	noConfig.Name, _ = types.NewIdentifier("noConfig")
 
-	noTags := sk
+	noTagName, _ := types.NewIdentifier("noTagSink")
+	noTags := sinks.Sink{
+		Name:        noTagName,
+		Description: "An example prometheus sink",
+		Backend:     "prometheus",
+		State:       sinks.Unknown,
+		Error:       "",
+		Config:      map[string]interface{}{"remote_host": "data", "username": "dbuser"},
+		Tags:        map[string]string{"cloud": "aws"},
+	}
+	noTags, err = service.CreateSink(context.Background(), token, noTags)
+	require.Nil(t, err, fmt.Sprintf("unexpected error: %s", err))
+	noTags.Backend = ""
 	noTags.Tags = make(map[string]string)
-	noTags.Name, _ = types.NewIdentifier("noTags")
 
 	noDescription := sk
-	noDescription.Tags = make(map[string]string)
-	noDescription.Name, _ = types.NewIdentifier("noDesc")
+	noDescription.Description = ""
+
+	newTagsSink := sk
+	newTags := types.Tags{"cloud": "aws", "test": "true"}
+	newTagsSink.Tags = types.Tags{"test": "true"}
+
+	newVTagName, _ := types.NewIdentifier("newKTagSink")
+	newVTagSink := sinks.Sink{
+		Name:        newVTagName,
+		Description: "An example prometheus sink",
+		Backend:     "prometheus",
+		State:       sinks.Unknown,
+		Error:       "",
+		Config:      map[string]interface{}{"remote_host": "data", "username": "dbuser"},
+		Tags:        map[string]string{"cloud": "aws"},
+	}
+	newVTagSink, err = service.CreateSink(context.Background(), token, newVTagSink)
+	require.Nil(t, err, fmt.Sprintf("unexpected error: %s", err))
+	newVTagSink.Backend = ""
+
+	newVTagSink.Tags = map[string]string{"cloud": "true"}
+	newVTag := types.Tags{"cloud": "true"}
```

Comment:
I think, better naming of variables will be a good thing here.
iex. newTags, newTagsSink, newVTagSink, etc.. 
Try using the use-case name of each, and some you can add directly in the case struct, like.
```golang
                   "update sink tags with new tags": {
			sink:  newTagsSink,
			tags:  types.Tags{"cloud": "aws", "test": "true"}, // Eliminating the need of newTags
			token: token,
			err:   nil,
		},
```